tests/test_paifu.py

```diff
@@ -0,0 +1,17 @@
+from unittest import TestCase
+
+from mahjong.record.reader import from_file
+
+
+class TestTile(TestCase):
+    def test_tile_from_tenhou(self):
+        a = from_file("tests/2009060321gm-00b9-0000-75b25bcf.xml")
+        retlen = []
+        for z in a.game_list:
+            for i in z.events:
+                ret = i.to_paifu()
+                retlen.append(len(ret))
+        assert retlen == [5, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
```

Comment:
This assertion assume that `INIT` must have been parsed to 5 events, which is conter-intutive and weird for developer and not flexable. Consider seperate it into another unit test (method), and name the new unit test properly (like `test_INIT_must_have_5_event`)